tests/integration/docker-compose.yml

```diff
@@ -16,18 +16,40 @@ services:
       - LOGOS_ORIGIN=null
       - BREACH_RESOLUTION_ENABLED=false
```

Comment:
Curious, should this be "true"? I think breach resolutions are enabled in production. Curious what effect this would have, apart from the logged in admin being slightly different.

Reply:
The breach detail pages will also be different for logged-in users.